xml/ha_clvm.xml

```diff
@@ -348,6 +345,139 @@ cLVM) for more information and details to integrate here - really helpful-->
   /dev/sdb vgtest b</screen>
     </step>
    </procedure>
+   <para>To test your clustered mirror device, proceed as follows:</para>
```

Comment:
Hi Tomas,
I feel we should just modify the problematic section, which is starting here:
"""
To support RAID10 for cmirrord, use the following procedure (assuming that /dev/sda and /dev/sdb are the shared storage devices): 
....
"""